pkg/models/api.go

```diff
@@ -17,10 +17,12 @@ package models
 
 // ComputedSecret holds all info about a secret
 type ComputedSecret struct {
-	Name          string `json:"name"`
-	RawValue      string `json:"raw"`
-	ComputedValue string `json:"computed"`
-	Note          string `json:"note"`
+	Name               string  `json:"name"`
+	RawValue           *string `json:"raw"`
+	ComputedValue      *string `json:"computed"`
+	RawVisibility      string  `json:"rawVisibility"`
+	ComputedVisibility string  `json:"computedVisibility"`
```

Comment:
It's been a while so I don't remember - when will the raw and computed visibilities diverge?

Reply:
If you have a non-restricted secret that references a restricted secret. (There's an example of this in the tables in the PR description if you want to see a real world example)

Reply:
Ah, right. Ty.